bin/p2-preparer/main.go

```diff
@@ -52,6 +56,14 @@ func main() {
 		if err != nil {
 			logger.WithError(err).Fatalln("Could not check for require file's existence")
 		}
+
+		required, err := loadTokens(preparerConfig.RequireFile)
+		if err != nil {
+			logger.WithError(err).Fatalln("Could not read required file")
```

Comment:
What do you think about including the require file path in this error log so that is more descriptive?